OMCompiler/Compiler/BackEnd/SymbolicJacobian.mo

```diff
@@ -94,11 +94,11 @@ public constant String DAE_CJ = "$DAE_CJ";
 
 public function symbolicJacobian "author: lochel
```

Comment:
Do you want to also change the name of these functions? At least, if they work for both ODE and DAE mode, their documentation should say that clearly

Reply:
If you search for "ODE" in this file, there are many instances of that...

Reply:
There is a dedicated function for DAEs:
https://github.com/OpenModelica/OpenModelica/blob/1711781d8ab93da9e5ba1c8466b537da28db838b/OMCompiler/Compiler/BackEnd/SymbolicJacobian.mo#L211

I really don't feel familiar enough with that part of the code to tell apart ODE and DAE stuff, so I am leaving it as it is. No harm done.